fpdc_client/base.py

```diff
@@ -40,17 +41,31 @@ class FPDC:
 
     app_id = "fpdc-client"
 
-    def __init__(self, url=DEFAULT_URL):
+    def __init__(self, url=PROD_URL):
         self.url = url
         self.schema = None
         self.client = coreapi.Client()
+        self.client_id = "fpdc-client"
+        self.client_secret = "notsecret"
+        if self.url is PROD_URL:
+            self.issuer = "https://id.fedoraproject.org/openidc/"
+        elif self.url is STG_URL:
+            self.issuer = "https://id.stg.fedoraproject.org/openidc/"
```

Comment:
There should probably be a `else` block here that defines `self.issuer` to `None`, otherwise the variable may be undefined in `login()` if no auth_file is passed, and it would crash.
The `login()` method should therefore make sure that `self.issuer is not None` and warn the user to create and pass an `auth_file` in that case.